### ISSUE TITLE
lfortran_intrinsics do not work with --fast

### ISSUE THREAD

**Issue by gptsarthak** [lcompilers/lpython#1698]
Currently any of the functions/imports that use the lfortran_intrinsics architecture do not work with the ```--fast``` option.
Functions that are supposed to return integer/float values, return 0 or garbage values.

For example:
```Python
import time
import random
import math

print(time.time())
print(random.randrange(10,20))
print(random.random())
print(math.atanh(0.90))
```
```console
(lp) sarthak@pop-os:~/lpython$ lpython examples/time.py
1.68124988955907702e+09
13
3.94382926819093038e-01
1.47221948958322035e+00
(lp) sarthak@pop-os:~/lpython$ lpython --fast examples/time.py
0.00000000000000000e+00
0
9.53282412436823800e-130
9.53282412436823800e-130

```
Update : The issue is somewhere in ```inline_function_calls``` pass.

**Comment by gptsarthak** [comment]
Fixed in #1693

**Comment by certik** [comment]
I think https://github.com/lcompilers/lpython/pull/1688 will address this issue as well.
